terraform/stacks/main/stack.tf

```
@@ -423,6 +419,16 @@ module "cloudwatch" {
   source = "../../modules/cloudwatch"
 
   stack_description = var.stack_description
-  sns_arn           = data.aws_sns_topic.cg_notifications.arn
+  sns_main_arn      = module.sns.sns_main_arn
+  sns_slack_arn     = module.sns.sns_slack_arn
   load_balancer_dns = module.cf.lb_arn_suffix
 }
+
+module "sns" {
+  source = "../../modules/sns"
+
+  sns_main_name      = var.sns_main_name
```

```suggestion
  sns_main_name      = "${var.stack_description}-platform-notifications"
```

This way, you don't need to pass in a variable from the outside or pipeline and it makes the name unique per environment, since `stack_description` will be `development`, `staging`, or `production`